docs/user/features.rst

```diff
@@ -88,4 +88,4 @@ Our bootstrapped company is owned and controlled by the founders,
 and fully funded by our customers and advertisers.
 That allows us to focus 100% of our attention on building the best possible product for you.
 
-Learn more :doc:`/about`.
+Learn more :doc:`/about/index`.
```

Comment:
Created a directory here.

It's not immediately in use. But the intention was to have an actual collection of articles, spanning some existing items once that we start doing path refactors and redirects (which we have said not to do right now)

Because of DirHTML, this shouldn't make a difference right now.